pkgs/tools/security/cve-bin-tool/default.nix

```
@@ -70,24 +43,38 @@ buildPythonApplication rec {
     rpmfile
     setuptools
     xmlschema
+    zipp
     zstandard
   ];
```

```suggestion
  ] ++ aiohttp.optional-dependencies.speedups;
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/aiohttp/default.nix#L91